app/controllers/vendor_api/vendor_api_controller.rb

```diff
@@ -10,6 +10,11 @@ class VendorApiController < ActionController::API
 
   private
 
+    def provider_visible_applications
```

Comment:
We already have `ApplicationChoice.for_provider` — could we push this scope onto the model and build on that? Others can benefit from it there too